### ISSUE TITLE
Bot didn't detect player's vehicle and player moving while in vehicle

### ISSUE THREAD

**Issue by ROBGUI09** [PrismarineJS/mineflayer#2394]
I wirtted a program which using pathfinder follows player, but bot stop following when i start to ride vehicles, i wanted to create script which auto mounts with player if he mounted boat, but playerentity.vehicle always return `undefined`. Is this a bug or unsupported?

**Comment by U5B** [comment]
What version of Minecraft are you attempting this on?

I do not know if mineflayer-pathfinder supports players in boats/minecarts/horses.

This might be a fix for Minecraft versions below 1.9. 
https://github.com/PrismarineJS/mineflayer/pull/2188

**Comment by ROBGUI09** [comment]
I am playing on 1.14.4 paper server. This is not pathfinder problem. The playerentity.vehicle is null when he riding something, and coordinates freezes at vehicle mount coordinate

**Comment by ROBGUI09** [comment]
Yes, if i use `bot.nearestEntity(e => e.type === "object" %% e.mobType === "Boat")` coordinates change properly, but i cant detect player in boat using `boatentity.passengers` too.
